client/my-sites/plans-v2/plans-column/index.tsx

```diff
@@ -61,7 +61,7 @@ const PlansColumn = ( { duration, onPlanClick, productType, siteId }: PlanColumn
 
 	return (
 		<div className="plans-column">
-			<FormattedHeader headerText={ translate( 'Plans' ) } brandFont />
+			<FormattedHeader headerText={ translate( 'Plans' ) } isSecondary={ true } brandFont />
```

Comment:
Non-blocking tiny detail, `isSecondary={ true }` can be `isSecondary` since by default props with no value will be true.